lib/crowbar/client/command/node/list.rb

```diff
@@ -73,6 +73,10 @@ def content_from(request)
                 )
               end
             end
+
+            result.sort do |x, y|
+              x.values.join <=> y.values.join
```

Comment:
If you tell me a source where you got that from we can talk about something different :)